documentation/3.0/course.md

```diff
@@ -42,6 +46,7 @@ Here is the list of [default content fields](../../schemas/3.0/course-text-prope
 |`continuing`|How to continue the studies after this course (i.e. advanced level etc.)|
 |`detailedCost`|Detailed information about pricing and what’s included in the price|
 |`technicalPrerequisites`|Technical requirements (i.e. computer, operating system)|
+|`*Training Course Content*`|Detailed information about course content (i.e. topics, modules, syllabus)|
```

Comment:
Default content fields cannot be modified. I understand your need but I think we need to use a different strategy for this.